android/titanium/src/java/org/appcelerator/titanium/TiApplication.java

```diff
@@ -133,6 +134,8 @@ public static void updateActivityTransitionState(boolean state)
 		}
 	}
 
+	public static long START_TIME_MS = 0;
```

Comment:
##### NOTE: I'm putting this here so we can easily access the start time elsewhere in our SDK.